streamer/src/nonblocking/quic.rs

```diff
@@ -53,8 +53,8 @@ use {
 };
 
 /// Limit to 500K PPS
-const MAX_STREAMS_PER_100MS: u64 = 500_000 / 10;
-const MAX_UNSTAKED_STREAMS_PERCENT: u64 = 20;
+pub const MAX_STREAMS_PER_100MS: u64 = 500_000 / 10;
+pub const MAX_UNSTAKED_STREAMS_PERCENT: u64 = 20;
```

Comment:
If we are using the MAX_UNSTAKED_STREAMS_PERCENT, I think it is easier to reason about to allow the callers to specify max_streams_per_100ms instead of max_unstaked_streams_per_100ms.